src/ProfilesList.json

```
@@ -1,4 +1,5 @@
 [
+  "rapid_killer_9.json",
```

```suggestion
  "rapid-killer-9.json",
```
Match it with the filename of your profile file you have created.